### ISSUE TITLE
Documentation of metrics for tf.layers models

### ISSUE THREAD

**Issue by tafsiri** [tensorflow/tfjs#680]
I think we can improve the documentation for metrics used in calls to model.compile and how they relate to what one might see in a fit callback. I've recently been trying them out and found a few challenges

- Knowing what the available metrics are, the docs for [compile](https://js.tensorflow.org/api/latest/#tf.Model.compile) only mention 'accuracy'. Knowing what other ones are available would be nice
- Mapping these metrics to what is passed to a callback like onBatchEnd. So far I've noticed that 'loss' is always present even if not explicitly mentioned in model.compile. I also noticed that when i pass 'accuracy' to model.compile, the actual keys in the callback are 'acc', (and sometimes 'val_acc' & 'val_loss' are present if i have a validation set). It would be nice to better describe what metrics are available in what callback
- Understanding how https://js.tensorflow.org/api/latest/#metrics.binaryAccuracy fit into the picture: Can these be used with model.compile etc. 

There could be a number of ways to approach this: writing a guide, updating the API docs, adding new section header information to the relevant parts (or all three). I'm happy to help with this if we know the general direction we want to take.

cc @caisq @ericdnielsen @bileschi

**Comment by caisq** [comment]
@tafsiri Let's look this together as a part of the website improvement.

**Comment by ManishAradwad** [comment]
Hello @tafsiri , I'd like to begin working on this issue. Can you plz point me in some direction so I can start?

**Comment by tafsiri** [comment]
@ManishAradwad great! I think a good place to start would be update [these docs](https://github.com/tensorflow/tfjs/blob/tfjs-layers-v1.7.0/tfjs-layers/src/engine/training.ts#L443) with a list of available metrics and the keys they show up in a Log object.

This [metrics map](https://github.com/tensorflow/tfjs/blob/06263085469d1149ac962ab9b2a7c0c1be6c79f3/tfjs-layers/src/metrics.ts#L120) or the test file for metrics might be a good place to start in tracking down available metrics. The [test file](https://github.com/tensorflow/tfjs/blob/7eb7ae5391ce3fd65582cb2208d86e0f5a44ebee/tfjs-layers/src/metrics_test.ts#L297) might also be a good place to look.

cc @caisq who may have other ideas of how to dive into this.

**Comment by ManishAradwad** [comment]
@tafsiri Thanks for the reply! I had some questions:
1. I didn't quite get the meaning of `the keys they show up in a Log object`? What log object are we talking about here?
2. In the [metricsMap](https://github.com/tensorflow/tfjs/blob/06263085469d1149ac962ab9b2a7c0c1be6c79f3/tfjs-layers/src/metrics.ts#L120), I noticed some metrics as `mse` and `MSE`. What exactly is the difference between two?
3. Should I directly update the `metrics` list in the [these docstrings](https://github.com/tensorflow/tfjs/blob/tfjs-layers-v1.7.0/tfjs-layers/src/engine/training.ts#L443) or should I specify them in some other manner?

**Comment by gaikwadrahul8** [comment]
Hi, @tafsiri 

Thank you for opening this issue for tracking purposes. Since this issue has been open for a long time, the code/debug information for this issue may not be relevant with the current state of the code base.

The TFJs team is constantly improving the framework by fixing bugs and adding new features. We suggest you try the latest TFJs [version](https://github.com/tensorflow/tfjs/releases/tag/tfjs-v4.11.0) with the latest compatible hardware configuration which could potentially resolve the issue. We can keep the issue open if it is still relevant. Please confirm if we need to keep the issue open.

Thank you for your support and cooperation.

**Comment by gaikwadrahul8** [comment]
Hi, @tafsiri 

We have not received any confirmation from you, so we are closing this issue now. If the issue is still relevant, please let us know, either we'll re-open this issue or please feel free to create new issue after trying with latest version of Tensorflow.js. Thank you!
